src/CoreBundle/CronJob/CronjobStateService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
```

Comment:
Please add the copyright header and run PHP-CS-Fixer.

Reply:
ok